dist/cms.js

```diff
@@ -151,12 +151,45 @@ function getWordPressContentType() {
   return content;
 }
 
+/**
+ * Obtains data about Interactivity API elements on the page..
+ *
+ * @returns {object} Object with fields `total` and `total_by_type`.
+ */
+function getInteractivityAPIRegionCounts() {
+  // Look for data-wp-interactive elements.
+  const interactiveRegions = document.querySelectorAll('[data-wp-interactive]');
+
+  // Count the regions by type.
+  const regionTypeCounts = [];
+  interactiveRegions.forEach( region => {
+    const type = region.getAttribute('data-wp-interactive');
+    regionTypeCounts[ type ] = ( regionTypeCounts[ type ] || 0 ) + 1;
+  })
```

Comment:
This grouping doesn't make sense to me. What would be the purpose of it? To my knowledge the `type` variable contains most likely a JSON string, so why would we want to group by a JSON string?

Based on https://github.com/HTTPArchive/custom-metrics/pull/113#discussion_r1550430061, I thought we were going to get the counts per type of Interactivity API attribute - though we could also do that in a separate PR.

I'm just questioning why we would want to get counts grouped by this attribute value?

Reply:
> Since the technical concept is called "namespace" by the API, we should probably name this here accordingly, something like regionCountByNamespace?

Yea, good point - I was having fun calling it a type, but it really is a namespace. Will update naming to use namespace instead of type.

Reply:
switched everything to `namespace`. hopefully plugins follow the core naming pattern for namespace even though it isn't required and we get nice strings that let us count usage and presence on the page, the current approach works well for core blocks anyway.

Reply:
I think we need to account for both formats, as that's how the API is defined. We can't assume the `data-wp-interactive` attribute always contains a namespace string, otherwise we get incorrect data. So I think this code will need to check if the attribute contains a JSON string that resolves to an object with `namespace` property, or simply a single (non-JSON) string, which would need to be interpreted as the namespace.

Reply:
makes sense, wish we had a live example of this use; in the meantime I will create a mocked version to test.

Reply:
I added a mocked up test region that uses the JSON format on the test page: https://interactivity-api.instawp.xyz/test-post/ - tested and new namespace as part of JSON was detected properly